routes/users/users.js

```diff
@@ -0,0 +1,42 @@
+const users = require("express").Router()
+
+let usersArr = [
+    {
+    username: "user1234",
+    email: "user1234@gmail.com",
+    activationDate: "January 1, 2010"
+    },
+    {
+    username: "user4321",
+    email: "user4321@yahoo.com",
+    activationDate: "December 21, 2019"
+    }
+]
+
+users.get("/", (req, res) => {
+    res.json(usersArr)
+})
+
+const addUser = (req, res, next) => {
+    usersArr.push(req.body)
+    res.json(usersArr)
```

Comment:
You are sending a `res.json` response twice, this would work on the frontend but it would throw an error on the backend